Examples/Examples/WorldScaleGeoTrackingExampleView.swift

```diff
@@ -0,0 +1,92 @@
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+import CoreLocation
+
+/// An example that utilizes the `WorldScaleGeoTrackingSceneView` to show an augmented reality view
+/// of your current location. Because this is an example that can be run from anywhere,
+/// it places a red circle around your initial location which can be explored.
+struct WorldScaleGeoTrackingExampleView: View {
+    @State private var scene: ArcGIS.Scene = {
+        // Creates an elevation source from Terrain3D REST service.
+        let elevationServiceURL = URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!
+        let elevationSource = ArcGISTiledElevationSource(url: elevationServiceURL)
+        let surface = Surface()
+        surface.addElevationSource(elevationSource)
+        surface.backgroundGrid.isVisible = false
+        surface.navigationConstraint = .unconstrained
+        let scene = Scene(basemapStyle: .arcGISImagery)
+        scene.baseSurface = surface
+        scene.addOperationalLayer(parcelsLayer)
+        return scene
+    }()
+    
+    /// Basemap opacity.
+    @State private var opacity: Float = 1
+    /// Graphics overlay to show a graphic around your initial location.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    /// The location datasource that is used to access the device location.
+    @State private var locationDataSource = SystemLocationDataSource()
+    /// A feature layer with San Bernardino parcels data.
+    private static var parcelsLayer: FeatureLayer {
+        let parcelsTable = ServiceFeatureTable(url: URL(string: "https://services.arcgis.com/aA3snZwJfFkVyDuP/ArcGIS/rest/services/Parcels_for_San_Bernardino_County/FeatureServer/0")!)
+        let featureLayer = FeatureLayer(featureTable: parcelsTable)
+        featureLayer.renderer = SimpleRenderer(symbol: SimpleLineSymbol(color: .cyan, width: 3))
+        return featureLayer
+    }
+    
+    var body: some View {
+        VStack {
+            WorldScaleGeoTrackingSceneView(locationDataSource: locationDataSource) { proxy in
+                SceneView(scene: scene, graphicsOverlays: [graphicsOverlay])
+                    .onSingleTapGesture { screen, _ in
+                        print("Identifying...")
+                        Task {
+                            let results = try await proxy.identifyLayers(screenPoint: screen, tolerance: 20)
+                            print("\(results.count) identify result(s).")
+                        }
```

Comment:
This will be better in a task view modifier that gets kicked off every time the screen is tapped. That is because if the user taps the screen very fast multiple times the task modifier will cancel the task before it. Here a bunch of tasks will get kicked off regardless if another one started.

Reply:
I'm not sure how a task modifier can be used with the onSingleTapGesture view modifier. Can you clarify this feedback? Thanks.

Reply:
You could possibly do it like this (I have not tested it):

```swift
@State private var tappedScreenPoint: CGPoint?

SceneView(scene: scene, graphicsOverlays: [graphicsOverlay])
    .onSingleTapGesture { screenPoint, _ in
       tappedScreenPoint = screenPoint
    }
    .task(id: tappedScreenPoint) {
        guard let tappedScreenPoint else { return }
        
        let results = try? await proxy.identifyLayers(screenPoint: tappedScreenPoint, tolerance: 20)
        print("\(results?.count ?? 0) identify result(s).")
    } as! SceneView
```

But I'm not sure if we should advertise that force casting to a SceneView to our users.

Reply:
I tested this code and it results in a crash when casting the task modifier to a scene view. The same logic is in `TableTopExampleView` and `FlyoverExampleView`, so maybe this logic can be refactored later and applied to all examples.

Reply:
Darn